BillTrack.Domain/Entities/Workday.cs

```diff
@@ -0,0 +1,10 @@
+namespace BillTrack.Domain.Entities;
+
+public class Workday : BaseEntity
+{
+    public DateOnly Date { get; set; }
+    public TimeOnly Hours { get; set; }
```

Comment:
`Hours` could be a `decimal`. `TimeOnly` is also acceptable. But it will be required later to convert it to a decimal to calculate a result of multiplication "Salary" and "Hours" for a month.
An employee could work partially (less, than expected hours count in a month) or more, than a 1 full time (overtimes).